.cqfdrc

```diff
@@ -0,0 +1,6 @@
+[project]
+org='seapath'
+name='sv-timestamp-analysis'
+
+[build]
+command='python3 sv_timestamp_analysis.py --help'
```

Comment:
What is the use of cqfd if it is only for printing help?

Reply:
cqfd is used to launch the tool. The command needs to be provided by the user, I can't write a flavor for that, because I don't know the name of the sv files that will be used.
However, I cannot provide a blank command to cqfd build part, so I propose to launch the help when nothing is provided.